report-testing-framework/report-testing/src/test/java/org/jembi/bahmni/report_testing/cdv_ts_reports/HivtmiIndicator1ByEntrypointsTests.java

```diff
@@ -0,0 +1,89 @@
+package org.jembi.bahmni.report_testing.cdv_ts_reports;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+import java.util.Map;
+
+
+import org.jembi.bahmni.report_testing.test_utils.BaseReportTest;
+import org.jembi.bahmni.report_testing.test_utils.models.ConceptEnum;
+import org.jembi.bahmni.report_testing.test_utils.models.TherapeuticLineEnum;
+import org.jembi.bahmni.report_testing.test_utils.models.VisitTypeEnum;
+import org.jembi.bahmni.report_testing.test_utils.models.ReportEnum;
+import org.jembi.bahmni.report_testing.test_utils.models.GenderEnum;
+
+import org.joda.time.LocalDate;
+import org.joda.time.LocalDateTime;
+import org.junit.Test;
+
+
+public class HivtmiIndicator1ByEntrypointsTests extends BaseReportTest {
```

Comment:
@tresorluz we need to add unit tests to test each entry point individually. We also need unit tests for edge cases, like for instance when
- no entry point has been selected
- HIV test date is outside the reporting period
- no result has been captured